plugins/env/test/samson_env/samson_plugin_test.rb

```diff
@@ -174,7 +174,9 @@ def protect_against_forgery?
     end
 
     describe 'project_form callback' do
-      let(:checkbox) { 'id="project_use_env_repo" />' }
+      let(:checkbox) { 'id="use_env_repo"' }
+      let(:dep_env_repo) { 'zendesk/test.git' }
```

Comment:
should not link to .git

Reply:
Done ... and maybe a check for this in the future when setting the Samson env for the repo or strip off the `.git` if it exists.